int32 index
geometry_msgs/Vector3 point
